report/test/renderer/report-renderer-test.js

```diff
@@ -26,6 +27,8 @@ describe('ReportRenderer', () => {
   let sampleResults;
 
   beforeAll(() => {
+    global.console.warn = jest.fn();
```

Comment:
tsc is no problem, it's been good since node 12, though in unit test code it also doesn't seem to matter much? `global === globalThis` and this can't ever run in a browser